litmus-portal/frontend/src/components/YamlEditor/styles.ts

```diff
@@ -40,87 +62,268 @@ const useStyles = makeStyles((theme: Theme) => ({
     borderRadius: 3,
-    backgroundColor: '#161616',
+    backgroundColor: theme.palette.editorBackground,
     boxSizing: 'border-box',
     color: theme.palette.secondary.contrastText,
-    borderColor: theme.palette.secondary.contrastText,
+    borderColor: 'rgba(255, 255, 255, 0.2)',
     paddingLeft: theme.spacing(3.125),
     width: '4rem',
     height: '2.75rem',
     marginLeft: theme.spacing(6.25),
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(1.5),
+    },
+    [theme.breakpoints.down('md')]: {
+      marginLeft: theme.spacing(1),
+    },
   },
 
   editorButtonReplace: {
     borderRadius: 3,
-    backgroundColor: '#161616',
+    backgroundColor: theme.palette.editorBackground,
     boxSizing: 'border-box',
     color: theme.palette.secondary.contrastText,
-    borderColor: theme.palette.secondary.contrastText,
+    borderColor: 'rgba(255, 255, 255, 0.2)',
     paddingLeft: theme.spacing(3.125),
     width: '4rem',
     height: '2.75rem',
     marginLeft: theme.spacing(15.625),
+    [theme.breakpoints.down('xs')]: {
+      marginLeft: theme.spacing(-0.5),
+      marginTop: theme.spacing(1.5),
+    },
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(1.3),
+    },
+    [theme.breakpoints.down('md')]: {
+      display: 'none',
+    },
+    [theme.breakpoints.down('lg')]: {
+      marginLeft: theme.spacing(5),
+    },
   },
 
   editorButtonSelectAll: {
     borderRadius: 3,
-    backgroundColor: '#161616',
+    backgroundColor: theme.palette.editorBackground,
     boxSizing: 'border-box',
     color: theme.palette.secondary.contrastText,
-    borderColor: theme.palette.secondary.contrastText,
+    borderColor: 'rgba(255, 255, 255, 0.2)',
     paddingLeft: theme.spacing(3.125),
     width: '4rem',
     height: '2.75rem',
     marginLeft: theme.spacing(7.5),
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(1.4),
+    },
+    [theme.breakpoints.down('md')]: {
+      marginLeft: theme.spacing(1),
+    },
+    [theme.breakpoints.down('lg')]: {
+      marginRight: theme.spacing(2.5),
+    },
+    [theme.breakpoints.down('xs')]: {
+      marginLeft: theme.spacing(1.5),
+      marginTop: theme.spacing(1.5),
+    },
+    [theme.breakpoints.down('xl')]: {
+      marginRight: theme.spacing(2),
+    },
   },
 
   editorButtonFullScreen: {
     borderRadius: 3,
-    backgroundColor: '#161616',
+    backgroundColor: theme.palette.editorBackground,
     boxSizing: 'border-box',
-    color: theme.palette.secondary.contrastText,
-    borderColor: '#161616',
+    color: 'rgba(255, 255, 255, 0.2)',
+    borderColor: theme.palette.editorBackground,
     paddingLeft: theme.spacing(3.75),
     paddingBottom: theme.spacing(1.875),
+    marginTop: theme.spacing(-2),
     width: '1.875rem',
     height: '1.875rem',
-    marginLeft: theme.spacing(1.25),
+    marginLeft: theme.spacing(-7.5),
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(-2),
+      width: '0rem',
```

Comment:
0 rem is not needed replace with width: 0,
```suggestion
      width: 0,
```